cmake/docs.cmake

```diff
@@ -0,0 +1,39 @@
+# ---- Dependencies ----
+
+find_package(Python3 REQUIRED)
```

Comment:
I don't know what's the minimum version [this script](https://github.com/mosra/m.css/blob/master/documentation/doxygen.py) supports.

Reply:
Alright, we can try to fix it later, I assume it works for you, and since it was not a feature before, let's bring it in and I can try to make it work for myself later.